uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="modal.closeModal()"><span aria-hidden="true">&times;</span><span
+    class="sr-only">Close</span></button>
+  <h4 class="modal-title">Leave Request</h4>
+</div>
+<div class="modal-body clearfix">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="chr_custom-select chr_custom-select--full">
+        <select>
+          <option value="1" selected>Vacation (15 days available)</option>
+          <option value="2">Medical</option>
+        </select>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <uib-tabset class="chr-tab__outer">
+
+        <uib-tab heading="Details">
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">No. of Days</label>
+                <div class="col-xs-8">
+                  <div class="btn-group chr-toggle btn-group-justified" data-toggle="buttons">
+                    <label class="btn active">
+                      <input type="radio" name="options" id="option1" autocomplete="off" checked> Multiple Days
+                    </label>
+                    <label class="btn">
+                      <input type="radio" name="options" id="option2" autocomplete="off"> Single Day
+                    </label>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">From Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
```

Comment:
About `.chr-datepicker`: this seems to me another instance of a custom component that doesn't really need to exist. What you are doing is styling `.uib-datepicker-popup`, so then just style that, instead of going through some custom classes